### PR TITLE
feat: consolidate templates

### PR DESCRIPTION
**Pull request by simboonlong**
- added ebl (converted to tw)
- added coo (converted to tw)
- add tests
- documented in storybook and `example:application`
- https://www.pivotaltracker.com/story/show/174942532

[consolidated-documents.zip](https://github.com/TradeTrust/generic-templates/files/5663200/consolidated-documents.zip)

Screenshot 2020-12-09 at 10 29 12 AM

### CODE REVIEW

**Comment by Nebulis** [comment]
get rid of other repos ?

**Comment by simboonlong** [comment]
yeah want to, after i shift them over

File: src/templates/BillOfLading/BillOfLadingTemplate.test.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { screen } from "@testing-library/dom";
+import { render } from "@testing-library/react";
+import React from "react";
+import { BillOfLadingTemplate } from "./BillOfLadingTemplate";
+import { BillOfLadingSample } from "./sample";
+jest.spyOn(HTMLCanvasElement.prototype, "getContext").mockImplementation();
+
+describe("bill of lading", () => {
```
Comment (dhemutton) [inline src/templates/BillOfLading/BillOfLadingTemplate.test.tsx:8]: Maybe name it 'Bill of Lading' so can follow the other test descriptors?

**Review by dhemutton — APPROVED** [review]
lgtm👍

File: src/templates/BillOfLading/BillOfLadingTemplate.test.tsx (hunk 1 above)
Comment (simboonlong) [reply src/templates/BillOfLading/BillOfLadingTemplate.test.tsx:8]: Screenshot 2020-12-09 at 1 10 32 PM

looks like linter will complain small case

**Pull request merged**